packages/encoding/src/index.ts

```diff
@@ -1,12 +1,63 @@
 /**
  * Code modified from https://github.com/github/webauthn-json/blob/e932b3585fa70b0bd5b5a4012ba7dbad7b0a0d0f/src/webauthn-json/base64url.ts#L23
  */
+export const DEFAULT_JWK_MEMBER_BYTE_LENGTH = 32;
+
 export function stringToBase64urlString(input: string): string {
   // string to base64 -- we do not rely on the browser's btoa since it's not present in React Native environments
   const base64String = btoa(input);
   return base64StringToBase64UrlEncodedString(base64String);
 }
 
+export function base64urlToBuffer(baseurl64String: string): ArrayBuffer {
```

Comment:
Aside from tests this isn't used anywhere

Reply:
(same as https://github.com/tkhq/sdk/pull/321#discussion_r1717251137)